dev/integration_tests/non_nullable/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+# Exclude lib/main.dart to avoid warnings about the non_nullable
+# experiment before it is enabled.
+analyzer:
```

Comment:
This file is also excluded for non-nullable features. Again, the file is built so risk is low